.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
```

Comment:
As we use Guzzle 6 as our testing client, which requires PHP 5.5, we can no longer test 5.4 (we’re dropping 5.3 support anyway). Is this reason enough to switch to Guzzle 5? It’s, of course, a temporary problem: PHP 5.4 will be [EOL in 3 months](http://php.net/supported-versions.php).